python/lsst/meas/algorithms/ingestIndexManager.py

```diff
@@ -21,6 +21,7 @@
 
 __all__ = ["IngestIndexManager", "IngestGaiaManager"]
 
+from ctypes import c_int
```

Comment:
I would suggest `import ctypes` and then using it explicitly below. It only shows up twice in the code, so it's not like a `from ...` import is really saving any typing.